pkg/build_zip.py

```diff
@@ -65,6 +66,12 @@ def main(args):
       dst_path = _combine_paths(args.directory, dst_path)
 
       entry_info = ZipInfo(filename=dst_path, date_time=ts)
+      st = os.stat(src_path)
```

Comment:
This is not a general solution. See the comment in #96.

Reply:
Thanks for reviewing! I've redone the change to use a mode passed in the BUILD files with a sensible default modeled after what is done in pkg_tar.